src/bp/ui-admin/src/telemetry/client.ts

```diff
@@ -0,0 +1,179 @@
+import axios from 'axios'
+import shared from 'botpress/shared'
+import { createHash } from 'crypto'
+import _ from 'lodash'
+import ms from 'ms'
+import uuid from 'uuid'
+
+import store from '../store'
+
+import { EventData, EventPackageInfoType, StoreInfoType, TelemetryPackage } from './telemetry_type'
```

Comment:
Since you are already in your "telemetry" folder, please rename the file "typings.d.ts" so it's consistent everywhere